README.md

```diff
@@ -84,9 +82,100 @@ But you'll still need to pick the built version (which ships with the package).
 
 ```javascript
 // with the classic require...
-let Kuzzle = require('kuzzle-sdk/dist/kuzzle.js')
+const { Kuzzle } = require('kuzzle-sdk/dist/kuzzle.js')
 // ... or with the new import directive.
-import Kuzzle from 'kuzzle-sdk/dist/kuzzle.js'
+import { Kuzzle } from 'kuzzle-sdk/dist/kuzzle.js'
+```
+
+## Protocols used
+Actuellement, le SDK supporte nativement 3 protocols: `http`, `websocket` et `socketio`.  
+
+The main reason behind this is that while Socket.IO offers better compatibility with older web browsers, our raw WebSocket implementation is about 20% faster
+
+Which protocol is used when you connect to Kuzzle depends on multiple factors:
+
+#### NodeJS
+
+The protocol used is always raw WebSocket for `websocket`.
+
+#### Web Browsers
+
+If you choose `websocket`, the SDK will first try to use raw WebSocket to connect to Kuzzle. If the web browser does not support this protocol, then the SDK falls back to Socket.IO.
```

Comment:
not true anymore: the application must now implement the fallback if we need it.
(maybe give a short example for that.)